Planner/qSlicerPlannerModuleWidget.cxx

```diff
@@ -1159,16 +1159,9 @@ void qSlicerPlannerModuleWidgetPrivate::previewCut(vtkMRMLScene* scene)
                    splitNode2, scene);
 
   //add to hierarchy
-  vtkNew<vtkMRMLSubjectHierarchyNode> splitNodeH1;
-  vtkNew<vtkMRMLSubjectHierarchyNode> splitNodeH2;
-  splitNodeH1->SetHideFromEditors(1);
-  splitNodeH2->SetHideFromEditors(1);
```

Comment:
@sjh26 @bpaniagua 

```
  splitNodeH1->SetHideFromEditors(1);
  splitNodeH2->SetHideFromEditors(1);
```

I think I need to add these lines back in. I don't know why I removed them.

Reply:
OK, I've looked at this in a bit more detail. Obviously, I can't just add lines 1164 and 1165 back, because they use splitNode**H**1 and splitNode**H**2, which were hierarchy nodes, and have been removed. Would it accomplish the same thing to call `SetHideFromEditors(1)` on the nodes, `splitNode{1,2}`? Would it work to call `SetItemDisplayVisbility` using the SH node and an appropriate ID?